Python/Python.sublime-syntax

```diff
@@ -232,27 +403,30 @@ contexts:
     - match: '='
       scope: keyword.operator.assignment.python
       push:
+        - meta_scope: meta.function.default-value.python
```

Comment:
I've been trying not to stack similarly named scopes (for instance here there will be `meta.function.parameters.python meta.function.default-value.python`). I'm not sure if there is a great solution to this or not.